src/stylesheets/blocks/_problem.scss

```diff
@@ -1,4 +1,94 @@
 .problem-section {
   background-color: color(mint-green);
+
+  @media (screen(lg)) {
+    padding-bottom: rem(75);
+  }
+
+  .-lg-width {
+    background-color: color(white);
+    padding-top: rem(40);
+    padding-bottom: rem(20);
+
+    @media (screen(lg)) {
+      border-radius: rem(4);
+    }
+  }
+
+  .lg-text {
+    color: color(warm-charcoal);
+
+    @media (screen(lg)) {
+      text-align: left;
+    }
+  }
+
+  .sm-sans-serif-text {
```

Comment:
This is probably okay for this Florence project, but on a bigger project, any future additions to this block that require the "small sans serif" treatment will have that top and bottom padding applied (which may not be what you want). Then you have to split classes, one utility class for the "text styles" and the other bem class for anything else. Just something to keep in mind if you end up running into it.